docs/content/docs/test/Snackbar.md

```diff
@@ -0,0 +1,72 @@
+# Motivation
+
+**Why do we need a custom component for snackbar testing?** \
+Snackbar nature is unpredictable: it appears and disappears in asynchronious way. \
```

Comment:
i see nothing unpredictable in snackbar, it just hides with delay

Reply:
Unpredictable means that we can't predict when it will appear or disappear
Because we have no API to watch that behaviour

Reply:
that's not what you wrote )